jmx/jmx.go

```diff
@@ -75,18 +76,22 @@ func (cfg *connectionConfig) command() []string {
 		c = []string{cfg.executablePath}
 	}
 
-	c = append(c, "--hostname", cfg.hostname, "--port", cfg.port)
-	if cfg.uriPath != "" {
-		c = append(c, "--uriPath", cfg.uriPath)
-	}
-	if cfg.username != "" && cfg.password != "" {
-		c = append(c, "--username", cfg.username, "--password", cfg.password)
-	}
-	if cfg.remote {
-		c = append(c, "--remote")
-	}
-	if cfg.remoteJBossStandalone {
-		c = append(c, "--remoteJBossStandalone")
+	if cfg.connectionURL != "" {
+		c = append(c, "--connectionURL", cfg.connectionURL)
+	} else {
+		c = append(c, "--hostname", cfg.hostname, "--port", cfg.port)
```

Comment:
Don't we need anyway to parse the port e.g. when we want to setup the Java System properties for connecting to JMX?